File: font_end/lib/models/delivery_method.dart
class DeliveryMethod {
  final String id;
  final String name;
  final String duration;
  final double price;
  final String logoUrl;

  DeliveryMethod({
    required this.id,
    required this.name,
    required this.duration,
    required this.price,
    required this.logoUrl,
  });

  factory DeliveryMethod.fromJson(Map<String, dynamic> json) {
    return DeliveryMethod(
      id: json['id'],
      name: json['name'] ?? '',
      duration: json['duration'] ?? '',
      price: (json['price'] as num?)?.toDouble() ?? 0.0,
      logoUrl: json['logoUrl'] ?? '',
    );
  }
}

File: back_end/src/main/java/com/nguyenquocbao/back_end/entity/DeliveryMethod.java
package com.nguyenquocbao.back_end.entity;

import jakarta.persistence.*;
import lombok.*;
import java.util.UUID;

@Entity
@Table(name = "delivery_methods")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class DeliveryMethod {
    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    private UUID id;

    private String name;
    private String duration;
    private Double price;
    private String logoUrl;
}
